src/display/network_utils.js

```diff
@@ -61,6 +61,13 @@ function extractFilenameFromHeader(getResponseHeader) {
   const contentDisposition = getResponseHeader('Content-Disposition');
   if (contentDisposition) {
     let filename = getFilenameFromContentDispositionHeader(contentDisposition);
+    if (filename.includes('%')) {
+      try {
+        filename = decodeURIComponent(filename);
+      } catch (ex) {
+        // URIError "Malformed URI", e.g. for "%AA.pdf"
```

Comment:
This looks like an incomplete copy of https://github.com/mozilla/pdf.js/blob/323b2eabcfc24e883e1a435828d94864bb46349f/web/ui_utils.js#L584-L587, but I don't think it matters too much here so just shortening the code to `} catch (ex) { }` would suffice here.

Reply:
thank you! I have changed my code.